Frontend/src/app/auth/pages/register/register.component.html

```diff
@@ -1 +1,70 @@
-<p>register works!</p>
+<nb-layout>
```

Comment:
Since nb-layout is in the app-component, you are able to remove it from all pages. you can remove the nb-layout, nb-layout-header, as well as the nb-layout-column components. the only thing that should be left is the contents inside the nb-layout-column.

Reply:
nb-layout has been removed. I understand what we're doing with the pages now, thanks!